frontend/src/metabase/static-viz/components/Gauge/utils.tsx

```diff
@@ -0,0 +1,177 @@
+import type { PieArcDatum } from "@visx/shape/lib/shapes/Pie";
```

Comment:
this file is a good candidate to be covered with unit tests

Reply:
That's fair. Initially, everything is kind of mashed together in a single file, and the implementation changes a lot. But this is settled, so it's probably a good time to bring in tests that I forgot. 👍